apps/aehttp/src/sc_ws_api.erl

```diff
@@ -4,7 +4,8 @@
 
 %TODO type -> opaque
 -opaque protocol() :: jsonrpc.
--type response() :: {reply, map()} | no_reply | stop.
+-type reply() :: {{error, atom()}, map()} | map().
```

Comment:
Is this confusing between `sc_ws_api_jsonrpc:process_request` and `sc_ws_api_jsonrpc:process_incoming`?

Reply:
Apologies - I had somehow missed the ` | [reply()]` in the patch. Correct.